ci_scripts/download_localized_strings_from_lokalise.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [[ -z $(which lokalise2) ]]; then
+    echo "Installing lokalise2 via homebrew..."
+    brew tap lokalise/cli-2
+    brew install lokalise2
+fi
+
+if [[ -z $(which recode) ]]; then
+    echo "Installing recode via homebrew..."
+    brew install recode
+fi
+
+API_TOKEN=$(fetch-password mobile/lokalise/token -q)
+PROJECT_ID=$(fetch-password mobile/lokalise/ios -q)
+LANGUAGES="da,de,en-GB,en,es-AR,es,fi,fr-CA,fr,it,ja,ko,nb,nl,nn-NO,pt-BR,pt-PT,ru,sv,tr,zh-HANS,zh-HK,zh-TW"
+# Chinese (Traditional) is zh-TW by default in lokalise so we have to map it to zh-Hant
```

Comment:
If you want you can change this in Lokalise

Click the flag, then `Language Settings`
![image](https://user-images.githubusercontent.com/6652623/78920852-e4052d80-7a48-11ea-84c4-ecafcc7a3c3c.png)

Change `Custom language code` to zh-Hant
![image](https://user-images.githubusercontent.com/6652623/78920812-d485e480-7a48-11ea-84e7-78902b38e7b6.png)

Reply:
Yeah I like that better. Updated in lokalise